.gitignore

```diff
@@ -1,3 +1,3 @@
 composer.lock
 vendor
-.idea
\ No newline at end of file
+.idea
```

Comment:
Please make the above image responsive: it's very much stretched on my 34" screen.